configure.ac

```diff
@@ -269,11 +269,14 @@ else
   AC_MSG_ERROR(no. Please install intltool package)
 fi
 
-# phantomjs
+# chromium - same as test/common/cdp.py
 
-AC_PATH_PROG([PHANTOMJS], [phantomjs], [], [$srcdir/node_modules/.bin$PATH_SEPARATOR$PATH])
-AM_CONDITIONAL([WITH_PHANTOMJS], [test -n "$PHANTOMJS"])
-AC_SUBST([PHANTOMJS])
+AC_PATH_PROGS([CHROMIUM], [chromium-browsr chrome], [], [$srcdir/node_modules/chromium/lib/chromium/chrome-linux/chrome$PATH_SEPARATOR$PATH])
```

Comment:
"browsr" typo

Reply:
I think we should rename the package! ;)